ecosystem/indexer-grpc/indexer-grpc-post-processor/src/processor_status_checker.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright © Aptos Foundation
+
+use crate::metrics::PROCESSOR_LAST_UPDATED_TIME_LATENCY_IN_SECS;
+use anyhow::{ensure, Result};
+use aptos_indexer_grpc_utils::constants::GRPC_AUTH_TOKEN_HEADER;
+use aptos_protos::indexer::v1::{raw_data_client::RawDataClient, GetTransactionsRequest};
+use chrono::{prelude::*, Local, NaiveDateTime};
+use futures::StreamExt;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::BTreeMap,
+    ops::Div,
+    sync::Arc,
+    time::{Duration, SystemTime, UNIX_EPOCH},
+};
+use tracing::info;
+
+const PROCESSOR_STATUS_CHECKER_WAIT_TIME_IN_SECS: u64 = 10;
+const GET_LATEST_PROCESSOR_STATUS: &str = "/api/rest/get_lastest_processor_status";
```

Comment:
can prob move these to `ProcessorStatusCheckerConfig`? or maybe just the first wait time